docs/x509/ocsp.rst

```diff
@@ -133,6 +133,21 @@ Creating Requests
             :class:`~cryptography.hazmat.primitives.hashes.SHA384`, and
             :class:`~cryptography.hazmat.primitives.hashes.SHA512` are allowed.
 
+    .. method:: add_extension(extension, critical)
+
+        .. note::
+
+            The only extension currently supported is
+            :class:`~cryptography.x509.OCSPNonce`.
```

Comment:
Is there a reason for this note? We don't list all the valid extensions in the other `add_extension` methods